lab-andrew/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This is a good start, you should make sure to add information to the "author" property and instead of using "echo \"Error: no test specified\" in "scripts" > "test" you can use "mocha" since that will run your tests.  That way when users enter the command `npm test` in your assignment root directory the tests will run.  While mocha is a simple command to run directly it is good to get in the habit of specifying the appropriate commands for "scripts" commands to use since it will simplify tasks for your users down the road when you have more complex testing procedures.